### ISSUE TITLE
Allow setting xaxis/yaxis as string

### ISSUE THREAD

**Issue by ahuang11** [holoviz/hvplot#1442]
Like xaxis="top" "bottom" etc
https://holoviews.org/user_guide/Customizing_Plots.html#axis-positions

**Comment by maximlt** [comment]
Would it also be OK for you to use an enum like `hvplot.AXIS_LOC.Top` (naming can be improved)?

**Comment by ahuang11** [comment]
Yeah I'm fine with that, as long it supports both (i.e. "top" and hvplot.AXIS_LOC.Top)

**Comment by maximlt** [comment]
@ahuang11 I think this was just a bug, MR to fix it https://github.com/holoviz/hvplot/pull/1497.

As for the enums, I'm not sure why I suggested it exactly now, I was probably using a library that offered options this way and thought it was a great user experience. I agree that should be implemented in HoloViews first.
